code/_compatibility/513.dm

```diff
@@ -1,9 +1,3 @@
-#if DM_VERSION > 513
-#warn Remie said that lummox was adding a way to get a lists
-#warn contents via list.values, if that is true remove this
-#warn otherwise, update the version and bug lummox
-#endif
-
```

Comment:
??????

Reply:
it _can_ be removed

Reply:
It is still off-top, and this warns should be checked